scripts/script.js

```diff
@@ -0,0 +1,176 @@
+//index.html & dogs.html click handlers for DOG PHOTOS
+
+function murphyInfo() {
+		alert("Murphy, Beagle, $123.45");
+}
+
+function poppyInfo() {
+		alert("Poppy, Shih Tzu, $123.45");
+}
+
+function jackInfo() {
+		alert("Jack, Jack Russel Terrier, $123.45");
+}
+
+function duffyInfo() {
+		alert("Duffy, Pitbull, $123.45");
+}
+
+function lucasInfo() {
+		alert("Lucas, Beagle, $123.45");
+}
+
+function jakeInfo() {
+		alert("Jake, Labrador, $123.45");
+}
+
+function angusInfo() {
+		alert("Angus, Boxer, $123.45");
+}
+
+function violetInfo() {
+		alert("Violet, Chocolate Labrador, $123.45");
+}
+
+function piperInfo() {
+		alert("Piper, Chocolate Labrador, $123.45");
+}
+
+function maximusInfo() {
+		alert("Maximus, Chocolate Labrador, $123.45");
+}
+
+function lunaInfo() {
+		alert("Luna, Labrador, $123.45");
+}
+
+function stellaInfo() {
+		alert("Violet, Chihuahua, $123.45");
+}
+
+//index.html & dogs.html click handlers for ADOPT BUTTONS
+let total = 0;
+
+function adoptAlert() {
+		total += 123.45;
+	alert('This dog has been added to your cart. Total: $' + total);
+}
+
+//More buttons
+function buttonMore () {
+	var dots = document.getElementById("initial-dog-cards");
+	var moreText = document.getElementById("more");
+  var btnText = document.getElementById("myBtn");
+
+  if (dots.style.display === "none") {
+    dots.style.display = "inline";
+    btnText.innerHTML = "Read more";
+    moreText.style.display = "none";
+  } else {
+    dots.style.display = "none";
+    btnText.innerHTML = "Read less";
+    moreText.style.display = "inline";
+  }
+}
+
+//Generate blog posts dynamically
+
+const el = document.getElementsByClassName('blog-grid');
+
+let blogArr = [];
+
+//Blog Entry 1
+const blogEntry1 = document.createElement('div');
```

Comment:
I like your methodic building of these. Nice job. Make sure you indent in everything that lives inside the loop